src/dali_backend.cc

```diff
@@ -157,7 +159,7 @@ TRITONSERVER_Error* DaliModel::Create(TRITONBACKEND_Model* triton_model, DaliMod
 }
 
 struct RequestMeta {
-  uint64_t compute_start_ns, compute_end_ns;
+  TimeInt compute_interval;
```

Comment:
You lost the information, that it's nanoseconds. If you don't feel like putting it in the variable name, maybe comment would be nice?

Reply:
I added a comment. It's very local helper class, so I don't think it requires a precise documentation. Also, now my timing functions have `_ns` suffix in them.